frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -463,7 +463,8 @@ common_settings.back_in_reader = {
         genGenericMenuEntry(_("Go to previous read page"), "back_in_reader", "previous_read_page"),
     },
 }
-if Device:hasKeyboard() then
+-- Kindle keyboard does not have a 'Backspace' key
```

Comment:
What's the current behavior of the Del button?

Reply:
On kindle, deletes the previous char. Elsewhere deletes the next char.